python/ql/test/library-tests/web/client/stdlib/ClientHttpRequests.ql

```diff
@@ -0,0 +1,11 @@
+import python
```

Comment:
I'm slightly confused by the test setup. If you have tests in both `2` and `3`, then why have tests in the shared test directory? Are these not just duplicating the testing that goes on in `2` and `3` separately?

Ah, no. I see the point now: the shared tests are testing the use of `six` as a compatibility layer. Still, it bugs me a bit that all three of these tests are essentially identical, except for the imports.

Reply:
I reduced it to 2 now, which is slightly better. I still want a test that explicitly makes sure that `six` works, so personally I don't see how we can reduce it to 1. If you have ideas, I'm all ears

Reply:
I don't think we can reduce it to a single test, as we want to test three different things, but we only get two invocations of the test runner to do so. I think the combined test + tests specific to `six` makes sense as a division.